operations/tempo-mixin/yamls/alerts.yaml

```diff
@@ -51,13 +51,24 @@
       sum by (cluster, namespace) (increase(tempodb_compaction_errors_total{}[5m])) > 0
     "labels":
       "severity": "critical"
+  - "alert": "TempoIngesterFlushesFailing"
+    "annotations":
+      "message": "Greater than 2 flushes have failed in the past hour, and retries have failed."
+      "runbook_url": "https://github.com/grafana/tempo/tree/main/operations/tempo-mixin/runbook.md#TempoIngesterFlushesFailing"
+    "expr": |
+      sum by (cluster, namespace) (increase(tempo_ingester_failed_flushes_total{}[1h])) > 2 and
+      sum by (cluster, namespace) (increase(tempo_ingester_failed_flushes_total{}[5m])) > 0 and
+      sum by (cluster, namespace) (increase(tempo_ingester_flush_failed_retries_total{}[5m])) > 0
```

Comment:
I think this file needs to be regenerated @joe-elliott

Reply:
yup, thx!

Reply:
+1. Normally CI should fail on this, but it seems that GitHub Actions didn't trigger?